chips/nrf52/src/lib.rs

```diff
@@ -1,6 +1,4 @@
-#![feature(
-    asm, concat_idents, const_fn, const_cell_new, try_from, core_intrinsics, iterator_step_by
-)]
+#![feature(asm, const_fn, core_intrinsics, iterator_step_by)]
```

Comment:
Is not `step_by` is stablized now?

Reply:
Not with this nightly, but I can try on a new nightly.

Reply:
Sorry, looked at the wrong release will definitely be in `1.28.0` but it was merged in for 23 days ago and should be available soon I guess!